pygmt/datasets/earth_magnetic_anomaly.py

```diff
@@ -18,6 +18,17 @@ def load_earth_magnetic_anomaly(
     r"""
     Load the Earth magnetic anomaly datasets in various resolutions.
 
+    +-----------------------------------+-----------------------------------+
```

Comment:
Better to use the `list-table` syntax (https://docutils.sourceforge.io/docs/ref/rst/directives.html#list-table).

The following may work:
```
.. list-table:: 
   :widths: 50 50
   :header-rows: 1

   * - Global Earth Magnetic Anomaly Model (EMAG2)
     - World Digital Magnetic Anomaly Map (WDMAM)
   * - .. figure:: https://www.generic-mapping-tools.org/remote-datasets/_images/GMT_mag.jpg
     - .. figure:: https://www.generic-mapping-tools.org/remote-datasets/_images/GMT_wdmam.jpg
```

Reply:
Thanks! I made this change in commit https://github.com/GenericMappingTools/pygmt/pull/2728/commits/5b17e6b4a2ae7c93dec2db0ee5f124e825e4bae4.